### PR TITLE
Add Mailer consumer for scheduled mails

### PR DESCRIPTION
**Pull request by ottaviano**
the test is coming

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -295,6 +295,13 @@ old_sound_rabbit_mq:
             callback:          AppBundle\Consumer\ProjectCitizenCreationNotificationConsumer
             qos_options:       { prefetch_size: 0, prefetch_count: 1, global: false }
 
+        mailer:
+            connection:        default
+            exchange_options:  { name: 'mails', type: direct }
+            queue_options:     { name: 'mails', durable: false, routing_keys: [mailer.scheduled_mail]}
```
Comment (B-Galati) [inline app/config/config.yml]: Why do we need to specify a routing key ? it could be anything no ?

File: src/Consumer/AbstractConsumer.php [hunk 2]
```diff
@@ -76,10 +76,15 @@ protected function getManager(): EntityManagerInterface
         return $this->manager;
     }
 
+    protected function allowExtraFields(): bool
```
Comment (B-Galati) [inline src/Consumer/AbstractConsumer.php]: `isExtraFieldsAllowed` would be a better name IMO.
This one looks an "action" and/or "behavior" if you know what I mean.

Comment (Remg) [reply src/Consumer/AbstractConsumer.php]: i know what you mean.

Comment (ottaviano) [reply src/Consumer/AbstractConsumer.php]: agree ;)

File: app/config/config.yml (hunk 1 above)
Comment (ottaviano) [reply app/config/config.yml]: removed

**Comment by thomasbisignani** [comment]
:medal_sports:

**Pull request merged**